vividus-plugin-mobile-app/src/main/java/org/vividus/selenium/mobileapp/MobileAppWebDriverManager.java

```diff
@@ -61,7 +62,7 @@ public <T> T getSessionDetail(String detail)
     @SuppressWarnings("unchecked")
     public int getStatusBarSize()
     {
-        if (isTvOS())
+        if (fullscreenApp)
```

Comment:
can't we check whether system status bar is displayed or not in fully automated way?

Reply:
I would say it is complicated in our particular case, the app running in fullscreen yet the status bar size is returned. (caps way)
I am not sure about searching via locator because it might be unstable and we have already declined this approach

Reply:
> the app running in fullscreen yet the status bar size is returned. (caps way)

what is the returned for your case?